src/aci_docs_sample.py

```diff
@@ -249,7 +243,7 @@ def run_task_based_container(aci_client, resource_group, container_group_name,
                            restart_policy=ContainerGroupRestartPolicy.never)
 
     # Create the container group
-    result = aci_client.container_groups.create_or_update(resource_group.name,
+    result = aci_client.container_groups.begin_create_or_update(resource_group.name,
```

Comment:
Instead of using sleep (), you can use the result() function to get the request result.